charts/influxdb/templates/backup-cronjob.yaml

```diff
@@ -128,8 +132,10 @@ spec:
             - /bin/sh
             args:
             - '-c'
+            - '-e'
             - |
               aws {{- if .Values.backup.s3.endpointUrl }} --endpoint-url={{ .Values.backup.s3.endpointUrl }} {{- end }} s3 cp --recursive "$SRC_URL" "$DST_URL"
+              rm -rf /backup/*
```

Comment:
No need to check if pushing backup to storage succeeded?

Reply:
Right, i guess it exists and tries if non zero return from aws.

Reply:
They have set `-e` to fix it. This is copied from the upstream repo.